logging_demo/test/test_logging_demo.py.in

```diff
@@ -24,11 +25,14 @@ import launch_testing_ros
 
 
 def generate_test_description():
+    env = os.environ.copy()
+    env['RCUTILS_CONSOLE_OUTPUT_FORMAT'] = '[{severity}] [{name}]: {message}'
     launch_description = LaunchDescription()
     process_under_test = ExecuteProcess(
         cmd=['@LOGGING_DEMO_MAIN_EXECUTABLE@'],
         name='test_logging_demo',
-        output='screen'
+        output='screen',
+        env=env
```

Comment:
Nitpick: adding a trailing comma here avoids touching this line when another keyword argument is appended in the future.

Reply:
Added.